### PR TITLE
Feature: Basic File List Page

### PR DESCRIPTION
**Pull request by YehorPytomets**
### Changes introduced by this PR:
* Add `List` for rendering `ListItemComponent`s inside.
* Add `ListItemModel`, which stores description of each item.
  * Descendants: `FileModel`, `FolderModel`.
* Add new components: `UserDetails`, `Breadcrumbs`, `Glyphicon`.
* Add `ListExplorer` page element which renders `List` and supplementary components inside.

### CODE REVIEW

**Comment by YehorPytomets** [comment]
@dkalinin , PR is ready to review.

File: webclient/app/components/breadcrumbs.js [hunk 1]
```diff
@@ -33,15 +33,26 @@ export default class Breadcrumbs extends Component {
...
+
+    if (parentFolderId) {
+      iconContainer.innerHTML = `
+        <a href="#/folder/${parentFolderId}" title="Back" data-type="breadcrumbs-parent-folder-id"></a>
+      `;
+      glyphiconContainer = iconContainer.firstElementChild;
+      glyphIconClass = 'glyphicon-level-up';
+    } else {
+      glyphiconContainer = iconContainer;
+      glyphIconClass = 'glyphicon-folder-open';
```
Comment (dkalinin) [inline webclient/app/components/breadcrumbs.js:50]: - [ ] Please refactor.

File: webclient/app/components/file-list/folder/folder-component.js [hunk 2]
```diff
@@ -39,12 +36,38 @@ export default class FolderComponent extends ListItemComponent {
           ${itemsAmount} ${itemsAmount === 1 ? 'item' : 'items'} 
         </td>
         <td class="table-col-actions" data-test="folder-component-actions">
-          <a title="Upload"><span class="glyphicon glyphicon-upload"></span></a>           
-          <a title="Remove" data-type="list-item-remove-button">
-            <span class="glyphicon glyphicon-remove-circle"></span>
-          </a>
+          <a title="Upload" data-type="list-item-upload-button"></a>           
+          <a title="Remove" data-type="list-item-remove-button"></a>
```
Comment (dkalinin) [inline webclient/app/components/file-list/folder/folder-component.js:40]: Please don't use `<a>` tag for button if it doesn't redirect.

**Comment by YehorPytomets** [comment]
@dkalinin , PR is ready to review.

File: webclient/app/components/file-list/file-list.js [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+import FileModel from '../../models/file-model.js';
+import FileType from './file/file-type.js';
+import FolderModel from '../../models/folder-model.js';
+
+/**
+ * Enumeration of possible type of list item.
+ *
+ * @type {Readonly<{FILE: string, FOLDER: string}>}
+ */
+const ListItemTypes = Object.freeze({
```
Comment (dkalinin) [inline webclient/app/components/file-list/file-list.js:13]: Object.freeze is not usually used.

**Comment by YehorPytomets** [comment]
@dkalinin , PR is ready to review.

**Pull request merged**